modules/jupyter-widget/package.json

```diff
@@ -18,6 +18,8 @@
   "files": [
     "dist/index.js",
     "dist/index.js.map",
+    "dist/standalone-html-bundle.js",
```

Comment:
Where is this file from?

Reply:
It no longer exists; I've cut it in the latest commit.